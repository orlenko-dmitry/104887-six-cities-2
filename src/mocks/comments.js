export default ([
  {
    comment: `The deluxe room was a quite comfortable one with all the adequate facilities. The only thing that made me feel uncomfortable was the rude behavior of an impolite staff at the reception desk.`,
    date: `2019-12-02T13:10:30.195Z`,
    id: 1,
    rating: 3,
    user: {
      avatarUrl: `https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/10.jpg`,
      id: 19,
      isPro: false,
      name: `Christina`,
    }
  },
  {
    comment: `I stayed here for one night and it was an unpleasant experience.`,
    date: `2019-12-02T13:10:30.195Z`,
    id: 2,
    rating: 2,
    user: {
      avatarUrl: `https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/2.jpg`,
      id: 11,
      isPro: false,
      name: `Jack`,
    }
  },
]);
