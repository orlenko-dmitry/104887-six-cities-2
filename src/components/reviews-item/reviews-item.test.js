import React from 'react';
import {shallow} from 'enzyme';

import ReviewsItem from '../reviews-item/reviews-item.jsx';
import comments from '../../mocks/comments.js';

it(`ReviewsItem renders correctly`, () => {
  const tree = shallow(<ReviewsItem review={comments[0]} key={comments[0].id}/>);

  expect(tree).toMatchSnapshot();
});
