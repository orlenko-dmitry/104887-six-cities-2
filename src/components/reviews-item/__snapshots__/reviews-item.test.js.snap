// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height={54}
        src="https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/10.jpg"
        width={54}
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Christina
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "60%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      The deluxe room was a quite comfortable one with all the adequate facilities. The only thing that made me feel uncomfortable was the rude behavior of an impolite staff at the reception desk.
    </p>
    <time
      className="reviews__time"
      dateTime="2019-12-02"
    >
      December 2019
    </time>
  </div>
</li>
`;
